style.css

```diff
@@ -0,0 +1,433 @@
+* {
+    box-sizing: border-box;
+  }
+
+/***  Font-Family: 'Roboto Condensed', Sans-Serif;
+  Font-Size: 35px;
+  Font-Family: 'Cabin', Sans-Serif;
+  Font-Size: 15px; ***/
+  
+html{
+    margin: 0;
+}
+
+.body{ 
+    font-family: 'Roboto Condensed', Sans-Serif;
+    margin: 0px;
+}
+
+a{
+    font-size: 20px;
+    color: #fbf0f0;
+    text-decoration: none;
+    padding: 50px; 
+    Font-Family: 'Roboto Condensed', Sans-Serif;
```

Comment:
Since you've already declared the font-family in the body-tag it will work on everything wrapped in the body - which is everything on your site. So you don't have to repeat it!